src/config.ts

```diff
@@ -274,6 +275,8 @@ export const DEFAULT_QUERY_CONFIG: QueryConfig = {
   // STYLIZE
   smallBandSizeForHighCardinalityOrFacet: {maxCardinality: 10, bandSize: 12},
   nominalColorScaleForHighCardinality: {maxCardinality: 10, palette: 'category20'},
+  xAxisOnTopForHighYCardinalityWithoutColumn: {maxCardinality: 30, orient: AxisOrient.TOP},
```

Comment:
The config already says on top, so I don't know why you need to customize `orient`.